_data/nav.yml

```diff
@@ -19,3 +19,15 @@ primary:
     href: /security-events/
   - text: Support
     href: /support/
+footer:
+  agencies:
+    become_a_partner: '$BASE_URL/partners/'
+    developer_guide: 'https://developers.login.gov'
+  learn:
+    about_us: '$LOCALE_BASE_URL/about-us/'
+    accessibility: '$LOCALE_BASE_URL/accessibility/'
+    join_us: '$LOCALE_BASE_URL/join/'
+    policies: '$LOCALE_BASE_URL/policy/'
+  support:
+    contact_us: '$LOCALE_BASE_URL/contact/'
+    help_center: '$LOCALE_BASE_URL/help/'
```

Comment:
does this variable work? I'd expect that we need to do like `| prepend: site.baseurl` in the templates

Reply:
I think this is based on how it's implemented in `identity-site`, but it requires an additional piece to replace the variable fragments:

https://github.com/GSA-TTS/identity-site/blob/ab27dfeb90459fddabe5258d32f90cb3b6adc3df/_includes/components/footer-navigation.html#L34

Since we don't need localization, maybe we can just drop the  "LOCALE_BASE_URL" bits?